### PR TITLE
Some accessibility fixes

### PR DESCRIPTION
**Pull request by cjcolvar**
This PR covers two issues:
- https://github.com/avalonmediasystem/avalon/issues/5566
- https://github.com/avalonmediasystem/avalon/issues/5567

Additional notes are in commit messages.

### CODE REVIEW

File: src/components/AutoAdvanceToggle/AutoAdvanceToggle.js [hunk 1]
```diff
@@ -19,14 +19,13 @@ const AutoAdvanceToggle = ({ label = "Autoplay", showLabel = true }) => {
           {label}
         </span>
       )}
-      <label className="ramp--auto-advance-toggle"
-        aria-labelledby='auto-advance-toggle-label'
-      >
+      <label className="ramp--auto-advance-toggle">
```
Comment (Dananji) [inline src/components/AutoAdvanceToggle/AutoAdvanceToggle.js:22]: In WAVE removal of `aria-labelledby` gave the following error;
accessibility-wave

**Review by Dananji — COMMENTED** [review]
I tested this with multiple accessibility checkers (WAVE, Firefox accessibility checker, and Siteimprove extension). And I got an in WAVE for the following.

File: src/components/AutoAdvanceToggle/AutoAdvanceToggle.js (hunk 1 above)
Comment (cjcolvar) [reply src/components/AutoAdvanceToggle/AutoAdvanceToggle.js]: I changed the label element to a div and SiteImprove and WAVE seem happy with it.

**Review by Dananji — APPROVED** [review]
👍

**Pull request merged**